app/models/abstract_feature.rb

```diff
@@ -0,0 +1,169 @@
+class AbstractFeature < ActiveRecord::Base
+  self.table_name = 'features'
+
+  class_attribute :lowres_simplification
+  self.lowres_simplification = 2 # Threshold in meters
+
+  belongs_to :spatial_model, :polymorphic => :true, :autosave => false
```

Comment:
Why is `autosave: false` necessary?

Reply:
See https://github.com/culturecode/spatial_features/commit/fe24163704e3c4ed1c780f7eaae8e8970831b87f